yarn-project/aztec-node/src/aztec-node/aztec-node.ts

```diff
@@ -1,16 +1,15 @@
-import { default as levelup } from 'levelup';
-import { default as memdown } from 'memdown';
 import { Archiver } from '@aztec/archiver';
+import { AztecAddress } from '@aztec/foundation';
 import { ContractData, L2Block, L2BlockSource } from '@aztec/l2-block';
+import { SiblingPath } from '@aztec/merkle-tree';
 import { P2P, P2PClient } from '@aztec/p2p';
-import { Tx, TxHash } from '@aztec/tx';
-import { MerkleTrees, WorldStateSynchroniser, ServerWorldStateSynchroniser, MerkleTreeId } from '@aztec/world-state';
 import { SequencerClient } from '@aztec/sequencer-client';
-import { AztecNodeConfig } from './config.js';
-import { SiblingPath } from '@aztec/merkle-tree';
-import { AztecAddress } from '@aztec/foundation';
-import { CircuitsWasm } from '@aztec/circuits.js';
+import { Tx, TxHash } from '@aztec/tx';
 import { UnverifiedData, UnverifiedDataSource } from '@aztec/unverified-data';
+import { MerkleTreeId, MerkleTrees, ServerWorldStateSynchroniser, WorldStateSynchroniser } from '@aztec/world-state';
+import { default as levelup } from 'levelup';
```

Comment:
Unrelated but we should banish this default as pattern, there is dedicated default import syntax